ui/packages/consul-ui/app/components/consul/auth-method/view/index.hbs

```diff
@@ -0,0 +1,217 @@
+              <li>
+                <CopyButton @value={{uri}} @name="Redirect URI" />
+                <span>{{uri}}</span>
+              </li>
+              {{/each}}
+            </ul>
+          </dd>
+        {{/if}}
+        {{#if @item.Config.BoundAudiences}}
+          <dt>{{t 'models.auth-method.Config.BoundAudiences'}}</dt>
+          <dd>
+            <ul>
+              {{#each @item.Config.BoundAudiences as |bond|}}
+              <li>
+                <span>{{bond}}</span>
+              </li>
+              {{/each}}
+            </ul>
+          </dd>
+        {{/if}}
+        {{#if @item.Config.OIDCScopes}}
+          <dt>{{t 'models.auth-method.Config.OIDCScopes'}}</dt>
+          <dd>
+            <ul>
+              {{#each @item.Config.OIDCScopes as |scope|}}
+              <li>
+                <span>{{scope}}</span>
+              </li>
+              {{/each}}
+            </ul>
+          </dd>
+        {{/if}}
+        {{#if @item.Config.JWTSupportedAlgs}}
+        <dt>{{t 'models.auth-method.Config.JWTSupportedAlgs'}}</dt>
+        <dd>{{join ', ' @item.Config.JWTSupportedAlgs}}</dd>
+        {{/if}}
+        {{#if @item.Config.VerboseOIDCLogging}}
+        <dt class="check">{{t 'models.auth-method.Config.VerboseOIDCLogging'}}</dt>
+        <dd><input type="checkbox" disabled="disabled" checked={{@item.Config.VerboseOIDCLogging}}></dd>
+        {{/if}}
+      {{/if}}
+      </dl>
+    </section>
+
+    <hr />
+
+    {{#if @item.Config.ClaimMappings}}
+    <section class="claim-mappings">
+      <h2>Claim Mappings</h2>
+      <p>Use this if the claim you are capturing is singular. When mapped, the values can be any of a number, string, or boolean and will all be stringified when returned.</p>
+      <table>
+        <thead>
+          <tr>
+            <td>Key</td>
+            <td>Value</td>
+          </tr>
+        </thead>
+        <tbody>
+          {{#each (entries @item.Config.ClaimMappings) as |entry|}}
+          <tr>
+            <td>{{get entry 0}}</td>
+            <td>{{get entry 1}}</td>
+          </tr>
+          {{/each}}
+        </tbody>
+      </table>
+    </section>
+    {{/if}}
+
+    <hr />
+
+    {{#if @item.Config.ListClaimMappings}}
+    <section class="list-claim-mappings">
+      <h2>List Claim Mappings</h2>
+      <p>Use this if the claim you are capturing is list-like (such as groups). When mapped, the values can be any of a number, string, or boolean and will all be stringified when returned.</p>
+            <table>
+        <thead>
+          <tr>
+            <td>Key</td>
+            <td>Value</td>
+          </tr>
+        </thead>
+        <tbody>
+          {{#each (entries @item.Config.ListClaimMappings) as |entry|}}
+          <tr>
+            <td>{{get entry 0}}</td>
+            <td>{{get entry 1}}</td>
+          </tr>
+          {{/each}}
+        </tbody>
+      </table>
+    </section>
+    {{/if}}
+  {{/if}}
+  </div>
```

Comment:
I think these two sections need empty states when they are empty. Would be good for another PR though.